packages/rancher-kiali-server/package.yaml

```diff
@@ -7,3 +7,4 @@ additionalCharts:
       templateDirectory: crd-template
       crdDirectory: templates
       addCRDValidationToMainChart: true
+doNotRelease: true
```

Comment:
Whoops, this should not be merged. Thanks @nickgerace for making me think of this with his comment on https://github.com/rancher/charts/pull/957#discussion_r597848601; we still need to release this chart to get the crd chart accessible for UI to install it before installing istio.

Reply:
No problem. Took me a year to scroll here, but happy to help ;)